### PR TITLE
Add !learn command and add beta docs to !hooks command

### PR DESCRIPTION
**Pull request by rahsheen**

### CODE REVIEW

File: src/features/commands.ts [hunk 1]
```diff
@@ -566,7 +566,10 @@ To integrate it into your editor: https://prettier.io/docs/en/editors.html`,
           The official React docs are the best resource for learning hooks:
           https://reactjs.org/docs/hooks-intro.html
 
-          However, the React docs still teach classes in the tutorials. A rewrite is in progress, but until then, there's a "React with Hooks" version of the React docs that uses hooks and function components for all examples:
+          However, the React docs still teach classes in the tutorials. A rewrite is in progress here: 
+          https://beta.reactjs.org
+
+          There's also a "React with Hooks" version of the React docs that uses hooks and function components for all examples:
```
Comment (ByronGuina) [inline src/features/commands.ts:572]: Can probably roll this comment into the above sentence.

> However, the React docs still teach classes in the tutorials. A rewrite focused on functional components and hooks is in progress here:

File: src/features/commands.ts [hunk 2]
```diff
@@ -577,6 +580,30 @@ To integrate it into your editor: https://prettier.io/docs/en/editors.html`,
...
+          title: "Learning React",
+          type: "rich",
+          description: `
+          The official (beta) React docs are the best resource for learning React:
+          https://beta.reactjs.org
+
+          The official React docs still teach classes for the examples, but the concepts are still valid:
+          https://reactjs.org/docs/getting-started.html
+
+          There's also a "React with Hooks" version of the React docs that uses hooks and function components for all examples:
```
Comment (ByronGuina) [inline src/features/commands.ts:599]: Same thing as the above comment.

> The official (beta) React docs are the best source for learning React. They focus on functional components and hooks:

Comment (rahsheen) [reply src/features/commands.ts:599]: It's something I wasn't sure about. My original thought was to just alias `!hooks` as `!learn`, but wasn't sure if others agreed there's kinda no reason to explicitly highlight hooks anymore.

Comment (ByronGuina) [reply src/features/commands.ts:599]: Yeah I think that makes sense to me. I think hooks have been the de facto mechanism for long enough, especially when the command is like 99% the same content just arranged differently.

**Review by BTMPL — APPROVED** [review]

**Pull request merged**